OpenRA.Mods.Common/Widgets/ScrollItemWidget.cs

```diff
@@ -57,6 +57,7 @@ public static ScrollItemWidget Setup(ScrollItemWidget template, Func<bool> isSel
 			w.IsVisible = () => true;
 			w.IsSelected = isSelected;
 			w.OnClick = onClick;
+			w.ItemKey = Convert.ToString(Game.CosmeticRandom.Next());
```

Comment:
Because ItemKey is a string? It is enough that the keys are unique, it doesn't matter what they are. What should I use as ItemKey then?